core/cloudflow-runner/src/main/scala/cloudflow/runner/LocalRunner.scala

```diff
@@ -112,25 +112,31 @@ object LocalRunner extends StreamletLoader {
 
     val streamlets = appDescriptor.streamlets.sortBy(_.name)
 
-    val streamletsWithConf = streamlets.zipWithIndex.map {
-      case (streamletDescriptor, idx) ⇒
-        val streamletName = streamletDescriptor.name
-        // Make sure that we convert any backslash in the path to a forward slash since we want to store this in a JSON value
-        val localStorageDirectory =
-          Files.createTempDirectory(s"local-runner-storage-${streamletName}").toFile.getAbsolutePath.replace('\\', '/')
-        log.info(s"Using local storage directory: $localStorageDirectory")
-        val deployment: StreamletDeployment =
-          StreamletDeployment(appDescriptor.appId, streamletDescriptor, "", idx, appDescriptor.connections)
-        val runnerConfigObj      = RunnerConfig(appId, appVersion, deployment, "localhost:" + kafkaPort)
-        val runnerConfig         = addStorageConfig(ConfigFactory.parseString(runnerConfigObj.data), localStorageDirectory)
-        val streamletParamConfig = streamletParameterConfig.atPath("cloudflow.streamlets")
-
-        val patchedRunnerConfig = runnerConfig
-          .withFallback(streamletParamConfig)
-          .withFallback(baseConfig)
-          .withValue("cloudflow.local", ConfigValueFactory.fromAnyRef(true))
-
-        (streamletDescriptor, patchedRunnerConfig)
+    var endpointIdx = 0
+    val streamletsWithConf = streamlets.map { streamletInstance ⇒
+      val streamletName = streamletInstance.name
+      // Make sure that we convert any backslash in the path to a forward slash since we want to store this in a JSON value
+      val localStorageDirectory =
+        Files.createTempDirectory(s"local-runner-storage-${streamletName}").toFile.getAbsolutePath.replace('\\', '/')
```

Comment:
Can we do this at the json encoding side? What are the issues if we escape it?

Reply:
I think it makes sense to make sure that the path is correct before putting it in JSON, but what you say could work (escaping). This is code that will likely get changed a lot with new local running, so I'll leave it like this for now.

Reply:
@RayRoestenburg what impact does this have for local running on Windows?

cc: @olofwalker

Reply:
This code has not changed guys.. we have tested local running on Windows.